frontend/src/components/AddWizard.tsx

```diff
@@ -8,23 +9,126 @@ type AddWizardProps = {
 export default function AddWizard(props: AddWizardProps) {
 
     const [name, setName] = useState<string>("")
+    const [gender, setGender] = useState<Gender | string>('')
+    const [attribute1, setAttribute1] = useState<Attribute1 | string>('');
+    const [attribute2, setAttribute2] = useState<Attribute2 | string>('')
+    const [attribute3, setAttribute3] = useState<Attribute3 | string>('')
```

Comment:
Is an empty string realy a good option as a initial value? Maybe you want to pre-select one option? Or set undefinded as initial value? `useState()`